requirements.txt

```diff
@@ -13,4 +13,5 @@ tiktoken
 py-cpuinfo
 gguf
 cmake
+auto-gptq
```

Comment:
why need gptq? which can only run on GPU

Reply:
@a32543254

Reply:
Yes, sync with zhenwei / weiwei , 
this module no need to add here